eva-server/src/main/java/uk/ac/ebi/eva/server/ws/FilesWSServer.java

```diff
@@ -52,8 +57,8 @@ public QueryResponse getFiles(@RequestParam("species") String species)
             throws IllegalOpenCGACredentialsException, IOException {
         initializeQueryOptions();
 
-        VariantSourceDBAdaptor variantSourceMongoDbAdaptor = DBAdaptorConnector.getVariantSourceDBAdaptor(species);
-        return setQueryResponse(variantSourceMongoDbAdaptor.getAllSources(queryOptions));
+        MultiMongoDbFactory.setDatabaseNameForCurrentThread(DBAdaptorConnector.getDBName(species));
+        return setQueryResponse(variantSourceEntityRepository.findAll());
```

Comment:
I think the old variantSourceMongoDbAdaptor.getAllSources returned a QueryResult, and this now is returning just a list of VariantSourceEntity. Use Utils.buildQueryResult